Source/AppLovinAdapter.swift

```diff
@@ -26,11 +27,7 @@ final class AppLovinAdapter: PartnerAdapter {
     let partnerDisplayName = "AppLovin"
     
     /// Instance of the AppLovin SDK
-    static var sdk: ALSdk? {
-        didSet {
-            AppLovinAdapterConfiguration.sync()
-        }
-    }
+    static var sdk = ALSdk.shared()
```

Comment:
can we recover this logic?
It seems like it was still used to sync the verboseLogging and locationCollection settings